apps/src/templates/rubrics/LearningGoal.jsx

```diff
@@ -64,7 +65,9 @@ LearningGoal.propTypes = {
 const AiToken = () => {
   return (
     <BodyFourText className={style.aiToken}>
-      {i18n.artificialIntelligenceAbbreviation()}
+      <ExtraStrongText>
```

Comment:
@bethanyaconnor please approve I haven't broke this 😄 
<img width="77" alt="image" src="https://github.com/code-dot-org/code-dot-org/assets/22244040/1119b398-e925-4f64-b319-6d608720fee1">


Once this pr is merged - you'll be able to simply wrap bodytext to `<StrongText></StrongText>` for `'medium'` body text and `<ExtraStrongText></ExtraStrongText>` for `'bold'` body text (I'll ask design team to update those 'medium' and 'bold' bodytext names so it'll correspond to Strong/ExtraStrong)

Reply:
That screenshot looks good! (Also that feature isn't launched yet so it's not a huge deal if it breaks -- we'll need to do some polish at some point anyway)

Reply:
fyi, I had to update this component a little in https://github.com/code-dot-org/code-dot-org/pull/53380. Let me know if you need any help with the merge conflicts!